src/bitcoin-cli.cpp

```diff
@@ -867,6 +868,71 @@ static void GetWalletBalances(UniValue& result)
     result.pushKV("balances", balances);
 }
 
+static void ParseGetInfoResult(UniValue& result)
```

Comment:
Could add Doxygen documentation for this new function.

Reply:
Added Doxygen documentation for `ParseGetInfoResult`

Reply:
👍 updated it according to your suggestion.